Userland/Applications/Screenshot/MainWindow.cpp

```diff
@@ -52,12 +52,27 @@ MainWindow::MainWindow()
 
     m_selected_area = *main_widget->find_descendant_of_type_named<GUI::RadioButton>("selected_area");
 
-    m_edit_in_pixel_paint = *main_widget->find_descendant_of_type_named<GUI::CheckBox>("edit_in_pixel_paint");
+    m_output_to_clipboard = *main_widget->find_descendant_of_type_named<GUI::RadioButton>("output_to_clipboard");
+
+    m_edit_in_pixel_paint = *main_widget->find_descendant_of_type_named<GUI::RadioButton>("edit_in_pixel_paint");
+
+    m_file = *main_widget->find_descendant_of_type_named<GUI::RadioButton>("file");
+
+    m_output_to_clipboard->on_checked = [this](bool is_checked) {
+        m_browse->set_enabled(!is_checked);
```

Comment:
I don't think the body here (or for `m_edit_in_pixel_paint`'s `on_checked`) is correct now. `m_browse` and `m_destination` should be enabled only if file output is enabled. If you toggle between pixel paint and clipboard, it currently also becomes incorrectly active, no?

Reply:
I think it works fine. I tested it thoroughly.

Proof:

![Screenshot](https://github.com/user-attachments/assets/cbbbe4ce-ff68-44ab-bf98-d0b9c8f875b2)

Reply:
Update: I simplified the code by only attaching a handler on the `on_checked` event of the `m_output_radio_file` widget. The code enables the `m_browse` and `m_destination` if the radio button is checked and disables them otherwise.